tests/conftest.py

```diff
@@ -35,3 +39,24 @@ def _get_fn(fn):
         return f"{test_dir}/data/{fn}"
 
     return _get_fn
+
+
+@pytest.fixture
+def gen_random_ptraj(request, tmp_path):
+    """
+    Fixture for preparing a temp location
```

Comment:
Docstring update, doesn't seem like the right description.

Reply:
All of that are leftover from testing. Made those changes :)

Reply:
Apparently scopes need to be per function (default) or it'll rewrite certain variables :)